### PR TITLE
chore: update go-asn1-ber to v1.5.5

### PR DESCRIPTION
**Pull request by cpuschma**
As follow up to the newest release of [go-asn1-ber v1.5.5](https://github.com/go-asn1-ber/asn1-ber/releases/tag/v1.5.5). Update the dependencies for both v3 and the project root

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -7,6 +7,8 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/go-asn1-ber/asn1-ber v1.5.4 h1:vXT6d/FNDiELJnLb6hGNa309LMsrCoYFvpwHDF0+Y1A=
 github.com/go-asn1-ber/asn1-ber v1.5.4/go.mod h1:hEBeB/ic+5LoWskz+yKT7vGhhPYkProFKoKdwZRWMe0=
```
Comment (johnweldon) [inline go.sum:9]: Fwiw, I think go mod tidy will remove this older version

Comment (cpuschma) [reply go.sum]: Agreed, forgot to ran "go mod tidy". Thank you!

**Review by johnweldon — APPROVED** [review]

**Comment by TomSellers** [comment]
@cpuschma / @johnweldon  - Do either of you have a sense for when a new version of the ldap package containing this change might be released? go-asn1-ber v1.5.5 fixes a few panics as does PR #455 in this repo. TIA.

**Comment by cpuschma** [comment]
Since there were quite a few changes since the last one, I think a new release is slowly due, but I want PR #460 to be merged first

**Pull request merged**